packages/fx-core/templates/plugins/resource/function/bicep/functionProvision.template.bicep

```diff
@@ -1,48 +1,79 @@
-param functionServerfarmsName string
-param functionAppName string
-param functionStorageName string
-{{#contains 'fx-resource-identity' Plugins}}
-param identityResourceId string
-{{/contains}}
+param provisionParameters object
```

Comment:
provisionParameters should has `@secure()` decorator

Reply:
It is fixed.